Prevents:
tests/libtest.c:1000:22: error: variable has incomplete type 'struct sockaddr_in'

--- a/tests/libtest.c
+++ b/tests/libtest.c
@@ -72,6 +72,8 @@ enum MHD_Result {
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <netdb.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
 
 #ifdef WITH_GNUTLS_IN_TESTSUITE
 #ifdef WITH_GNUTLS_OCSP
